AAI/AAIConnectProfile.md

```diff
@@ -99,12 +114,20 @@ described herein, encoded via JWS Compact Serialization per
 `ga4gh_passport_v1` as a space-separated entry within the `scope` claim but
 does not contain [GA4GH Claims](#term-ga4gh-claim).
 
-<a name="term-visa-issuer"></a> <a name="term-embedded-token-issuer"></a>
-**Visa Issuer** (aka "Embedded Token Issuer") -- a service that signs
-[Visas](#term-visa).
+<a name="term-passport"></a> **Passport** -- A signed and verifiable JWT container for holding [Visas](#term-visa).
+
+<a name="term-passport-issuer"></a> **Passport Issuer** --
+a service that creates and signs a [Passport](#term-passport-token) - in this case a JWT - holding [Visas](#term-visa).
+* a service where a Broker, Client or Claims Clearinghouse may re-sign the JWT holding GA4GH Claims (such as [GA4GH Visas](https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/ga4gh_passport_v1.md#passport-visa)) with their own authority.
 
-<a name="term-visa"></a> <a name="term-embedded-token"></a>
-**Visa** (aka "Embedded Token") -- A [GA4GH Claim](#term-ga4gh-claim)
+<a name="term-visa-issuer"></a>
+**Visa Issuer** -- a service that signs
+[Visas](#term-visa). This service:
+* may be a [Broker](#term-broker) itself
+* may be used by a [Broker](#term-broker) as part of collecting GA4GH Claims that the Broker includes in responses from its `/userinfo` or `/token` endpoint.
```

Comment:
I agree that userinfo endpoint may not be at `/userinfo` path, it can be on any URL that is specified in `/.well-known/openid-configuration` metadata under the key **"userinfo_endpoint"**. Thus using `'/userinfo` instead of userinfo endpoint is convenient, but imprecise.

Reply:
Userinfo Endpoint

Reply:
Will change to Userinfo Endpoint and Token Endpoint and add those to Terminology.